src/mesh.cpp

```diff
@@ -1104,18 +1104,14 @@ int CylindricalMesh::set_grid()
                "cylindrical meshes must start at phi >= 0.");
     return OPENMC_E_INVALID_ARGUMENT;
   }
-  if (grid_[1].back() > 360) {
+  if (grid_[1].back() > 2.0 * PI) {
     set_errmsg("phi-grids for "
-               "cylindrical meshes must end with theta <= 360 degree.");
+               "cylindrical meshes must end with theta <= 2*pi.");
```

Comment:
```suggestion
               "cylindrical meshes must end with theta <= 2π.");
```

Reply:
The underlying error message `openmc_err_msg` is just an array of `char`, so I don't think putting a unicode character in there will work

Reply:
Oh whoops, yeah agreed. I was on auto-pilot changing those.